### DISCUSSION TITLE
Help with block and container

### DISCUSSION THREAD

**Discussion by trinhhoanglong2000** [GrapesJS/grapesjs#4143] [Q&A]
So i recently start learning Grapejs
I want to create a div component that have another childrent component like this
![image](https://user-images.githubusercontent.com/51038586/153899557-0bf11a24-0443-4bd4-8265-cc73c02185e0.png)
my problem is when i add components props together with contents props, the content props is not showing anything
i want insde "dadada" there will be childrent that contains in components props
Thanks

**Comment by artf** [comment]
You should not mix `components` and `content` properties in the same component definition. The component `content` property is actually quite rare to use, are you sure you need it?

**Reply by trinhhoanglong2000** [reply]
thanks for replying!
my desired content would be
<div class="container">
        <div class="row">
            <div class="col-md-12">
              //====| component 1 |=====
              //====| component 2| =====
            </div>
   </div>
 </div>
if i'm going for the normal approaches, which is define each div as a component. It will result in having too many components in layer
What i would want is in layer there will be 3 component
A div, then 2 children inside of it only
I have try to add all of them and set layerable: false in which component i don't want to use but it will lead to all the childrent of that "layerable: false" to be disable from the layer also

**Reply by skru** [reply]
Is layerable meant to propagate down through the components @artf? If so got any tips on how to achieve setting layerable:false on a parent with children with layerable:true?

Sorry I can't help trinhhoanglong2000, I have the same issue, but yeh the components will override the content if set.
